Zend/zend_ast.c

```diff
@@ -990,6 +990,88 @@ ZEND_API zend_result ZEND_FASTCALL zend_ast_evaluate_inner(
 			}
 			return SUCCESS;
 		}
+		case ZEND_AST_CALL:
+		case ZEND_AST_STATIC_CALL:
+		{
+			zend_function *fptr;
+			switch (ast->kind) {
+				case ZEND_AST_CALL: {
+					ZEND_ASSERT(ast->child[1]->kind == ZEND_AST_CALLABLE_CONVERT);
+
+					zend_string *function_name = zend_ast_get_str(ast->child[0]);
+					fptr = zend_fetch_function(function_name);
```

Comment:
This function depends on lowercase function names and will thus break. Furthermore, it will not implement the default local ns lookup with a global fallback.